### PR TITLE
Embedding images

### PR DESCRIPTION
**Pull request by alishabab**
We have completed Youtube clone project with HTML and CSS.
And also checked for HTML validation and CSS linting errors.

### CODE REVIEW

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

File: index.html [hunk 1]
```diff
@@ -3,7 +3,487 @@
...
+        <div class="youtubeIcon">
+          <img src="./images/YoutubeIcon.PNG" alt="youtube icon" />
+        </div>
+        <div class="searchInput">
+          <div class="inputSection">
+            <input type="text" placeholder="Search" />
+          </div>
+          <div class="icon">
+            <img src="./images/searchIcon.PNG" alt="Search" />
+          </div>
```
Comment (addod19) [inline index.html]: Kindly replace this ```<div class="icon">``` with ```<button>```
n1

File: index.html [hunk 2]
```diff
@@ -3,7 +3,487 @@
...
+            <img src="./images/notification_youtube.PNG" alt="notification icon"  />
+          </section>
+          <section class="profile">
+            <img src="./images/Profile.jpg" alt="profile icon"  />
+          </section>
+        </div>
+      </div>
+    </nav>
+    <div class="container">
+      <div class="leftSection">
```
Comment (addod19) [inline index.html]: Let us improve semantics by using ```<main>``` instead of ```<div class="leftSection">```

File: index.html [hunk 3]
```diff
@@ -3,7 +3,487 @@
...
+          <div class="linkSection">
+            <img src="images/dropDown.PNG" alt="drop down" />
+            <a href="#">
+              View 63 replies
+            </a>
+          </div>
+        </section>
+      </div>
+      <div class="generalRightSection">
+        <div class="rightSect">
```
Comment (addod19) [inline index.html]: Let us improve semantics by using ```<aside>``` instead of ```<div class="generalRightSection">```

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes 🔴

Hi @alishabab and @OdongAlican you have done a great job 👍 , However, we need to make a few changes to get the project approved. Find comment below:
- Kindly fix the warnings returned by [HTML markup] (https://validator.w3.org/). They are easily fixable, declaring the ```<h6>`` and hiding it will solve this in the ```<section>``` parts.
- Kindly use ```<button>``` instead of ``````
- The project should have a descriptive README file. Kindly edit it and make the readme look professional [link](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/)
[OPTIONAL]
You can add padding between the image and the text in the ```rightSideSection```
n4
```Original```
n5

Kindly make these changes and submit code review 🙂 
Good luck with your next submission 😄 
Happy coding 💻 
Daniel Larbi Addo

**Comment by addod19** [comment]
For README reference check this [link](https://github.com/microverseinc/readme-template)

File: index.html (hunk 1 above)
Comment (alishabab) [reply index.html]: Replaced `<div class="icon">` with `<button>` tag.
` <button class="icon">
            Search
          </button>`

File: index.html (hunk 2 above)
Comment (OdongAlican) [reply index.html]: Replaced `<div class="leftSection">` with `<main>`

File: index.html (hunk 3 above)
Comment (alishabab) [reply index.html]: Replaced `<div class="generalRightSection">` with `<aside>`

**Review by euqueme — APPROVED** [review]
> ### Pull Request under the review by Maria Eugenia Queme P. (TSE)

### Hello @alishabab and @OdongAlican  :balloon:  :lollipop: 

### Thank you for doing all the changes requested by the previous reviewer, your project looks professional and this is the best Youtube clone I've reviewed so far :no_good_woman:  so you should feel proud of yourselves :1st_place_medal: 

[OPTIONAL] I have put an optional suggestion in your README if you want to address it :arrow_double_down: 

## that's it, team! :raised_hands: 

## please :pray:  keep up the good work! and submit a project completion form because your project is approved! cheers! :tada: :clinking_glasses: 

Happy coding! :woman_technologist: 

Any question? reach to me on slack **@ Maru**

PROJECT STATUS: APPROVED :green_apple:

File: README.md [hunk 4]
```diff
@@ -1,2 +1,63 @@
...
+- CSS
+
+## Live Demo
+
+[Live Demo Link](https://raw.githack.com/OdongAlican/Embedding-images-and-videos/embedding-images/index.html)
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
```
[the remark this answers is not in the document]
Comment (alishabab) [reply README.md]: Thank you @euqueme for the appreciation. We have deleted the template instructions from README.md

[the remark this answers is not in the document]
Comment (OdongAlican) [reply README.md]: Thanks a lot, @euqueme. Thank you for your guidance.

**Pull request merged**